docs/DeveloperGuide.md

```diff
@@ -790,3 +790,20 @@ testers are expected to do more *exploratory* testing.
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Effort**
+
+* As we were building off of AB3, we were able to reuse the model class and some of the commands.
+* In the beginning of our project (v1.2), this saved us time and effort as it was easy to build an MVP from an already
+functioning product (AB3).
+* However, challenges arose when we wanted to refine our MVP features to better suit ResearchRoster (v1.3)
+and add new features unlike what AB3 had (v1.3, v1.4, v1.5).
+* This was because we had to carefully look through the AB3 code to fully understand how it worked,
+in order to figure out how we can modify it, work with it, or work around it to implement changes and new features.
+* This resulted in us putting in a high amount of effort to maintain good OOP and use/build off the existing classes
+and methods in AB3, rather than simply creating new classes, methods and solutions that may have overlapped with
+existing classes and methods.
+* The high effort was also because some of the existing classes methods posed limitations to what we wanted to 
```

Comment:
For instance:
Edit command class and EditPersonDescriptor class needed lots of reworking because the initial functionality only allowed to overwrite list of tags whilst our product aimed to append to the list and delete from the list instead of resetting the whole list